modules/end-user-guide/partials/proc_building-custom-intellij-editor-images.adoc

```diff
@@ -0,0 +1,154 @@
+[id="building-custom-intellij-editor-images_{context}"]
+= Building custom IntelliJ editor images
+
+This procedure describes how to build an IntelliJ-based editor image using a custom archive and a custom image name.
+
+.Prerequisites
+
+* The `git`, `jq`, and `wget` tools are available.
+
+* Docker version 18.09 or greater, supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit].
+
+* Java Development Kit (JDK) version 11 is available.
+
+* On the macOS platform: the `+gnu-getopt+` tool is available.
```

Comment:
`available` seems both redundant and vague (this is just a list of required items, they don't need to be available to be on the list). Does the `+gnu-getopt+` tool need to be installed locally?